src/components/EmailValidationForm.tsx

```diff
@@ -1,71 +1,88 @@
 import * as React from "react";
-import { Form } from "library-simplified-reusable-components";
+import { Form, Button } from "library-simplified-reusable-components";
 import { connect } from "react-redux";
 import ActionCreator from "../actions";
 import { FetchErrorData } from "opds-web-client/lib/interfaces";
 import { Store } from "redux";
 import { State } from "../reducers/index";
 import { LibraryData } from "../interfaces";
-import { CheckSoloIcon } from "@nypl/dgx-svg-icons";
+import { CheckSoloIcon, XIcon } from "@nypl/dgx-svg-icons";
 
 export interface EmailValidationFormState {
-  sent: boolean;
+  validated: boolean;
 }
 
 export interface EmailValidationFormOwnProps {
-  library: LibraryData;
+  email: string;
+  libraryInfo: {};
+  uuid: string;
   store: Store<State>;
-  fetchLibrary: (uuid: string) => LibraryData;
-}
-
-export interface EmailValidationFormDispatchProps {
-  validateEmail: (data: FormData) => Promise<void>;
 }
 
 export interface EmailValidationFormStateProps {
   error?: FetchErrorData;
 }
 
+export interface EmailValidationFormDispatchProps {
+  validateEmail: (data: FormData) => Promise<void>;
+}
+
 export interface EmailValidationFormProps extends EmailValidationFormOwnProps, EmailValidationFormDispatchProps, EmailValidationFormStateProps {}
 
 export class EmailValidationForm extends React.Component<EmailValidationFormProps, EmailValidationFormState> {
-
   constructor(props) {
     super(props);
-    this.sendEmail = this.sendEmail.bind(this);
-    this.state = { sent: false };
+    this.validate = this.validate.bind(this);
+    this.renderInfoText = this.renderInfoText.bind(this);
+    this.state = { validated: false };
   }
 
-  async sendEmail(data: FormData): Promise<void> {
-    await this.props.validateEmail(data);
-    await this.props.fetchLibrary(this.props.library.uuid);
-    this.setState({ sent: !this.props.error });
+  convert(email: string): string {
```

Comment:
Why is this needed? It seems like it's manipulating emails from the server?

Reply:
No, it's converting the name of the email, not the actual email address.  Like, turning "contact_email" into "contact email" and vice versa.

Reply:
I see. Maybe `convertEmailTitle` would be better or something like that?